src/CommunityToolkit.Maui.Core/Platform/iOS/MCTPopup.macios.cs

```diff
@@ -0,0 +1,176 @@
+﻿using CommunityToolkit.Maui.Core;
+using Microsoft.Maui.Handlers;
+using Microsoft.Maui.Platform;
+using UIKit;
+
+namespace CommunityToolkit.Core.Platform;
+
+/// <summary>
+/// The navite implementation of Popup control.
+/// </summary>
+public class MCTPopup : UIViewController
```

Comment:
Let's avoid using `MCT` in `CommunityToolkit.Maui` and `CommunityToolkit.Maui.Core`. 

Long story short, marketing has strict rules around branding, and "MAUI Community Toolkit" always needs to be referred to as ".NET MAUI Community Toolkit". I agree it's silly, but renaming this is easier than causing a ruckus internally. 

This also now introduces two `Popup*` classes in `CommunityToolkit.Maui.Core`. 

Here's my recommendation:
- Rename this to `Popup`
- Rename `CommunityToolkit.Maui.Core.Views.Popup` to `CommunityToolkit.Maui.Core.View.Alert` since it's exclusively used for `IAlert`